src/_layouts/press.html

```diff
@@ -0,0 +1,28 @@
+---
+layout: default
+---
+
+{% if page.external %}
+  {% include script_redirect.html location=page.external %}
+{% endif %}
+
+{% capture date %}
+  {% include date.html date=page.date format = "%B %e, %Y, " %}
+{% endcapture %}
+
+{% assign location = page.location | default: "Sacramento, CA" %}
+
+{% unless page.external %}
+  <article class="press-release">
+    <a href="/press">Press</a>
+    <h1 class="h2">{{ page.heading }}</h1>
+    {% if page.lead %}
+      <p>{{ page.lead }}</p>
+    {% endif %}
+    <hr/>
+    <p class="immediate-release">For immediate release</p>
+    <p>{{ date | append: location | append: "—" | append: page.intro }}</p>
+    {{ page.content }}
+    <p class="end">###</p>
```

Comment:
What is this for?

Reply:
It's usually at the bottom of every Press Release, e.g. https://www.calitp.org/assets/Cal-ITP.Benefits.PressRelease.220921.pdf
![image](https://github.com/cal-itp/calitp.org/assets/1783439/5a1ed203-8a3e-4a91-8b12-b98c8bdf4dc7)

I see that it didn't make it into the design, so we can probably omit from the template.

Reply:
Oh I see. Thanks for linking that example